triedb/pathdb/nodes.go

```diff
@@ -424,3 +427,249 @@ func (s *nodeSetWithOrigin) decode(r *rlp.Stream) error {
 	s.computeSize()
 	return nil
 }
+
+// encodeNodeCompressed encodes the trie node differences between two consecutive
+// versions into byte stream. The format is as below:
+//
+// - metadata byte layout (1 byte):
+//
+//	┌──── Bits (from MSB to LSB) ───┐
+//	│ 7 │ 6 │ 5 │ 4 │ 3 │ 2 │ 1 │ 0 │
+//	└───────────────────────────────┘
+//	  │   │   │   │   │   │   │   └─ FlagA: marks if value is encoded in compressed format (1 always)
```

Comment:
what's the point of setting this bit if it's always true?

Reply:
OK, the comment is misleading. It means this flag is always true for `encodeNodeCompressed`. I will change it.